src/components/pages/Dashboard/SampleItem.js

```diff
@@ -1,21 +1,29 @@
-import React from 'react';
+// import React from 'react';
+// import styled from 'styled-components';
 
-export default props => {
-  // props
-  const { userData } = props;
+// const StyleCard = styled.div`
+// background:cyan;
+// `;
 
-  // condition for the component to render - this might be because of the state of user data or something else
-  function shouldRender() {
-    return userData;
-  }
+// export default props => {
+//   // props
+//   const { userData } = props;
 
-  return shouldRender() ? (
-    <div>Value if the component is available.</div>
-  ) : (
-    <div>
-      Value if the component is not available.
-      {/* Normally blank but might have a step the user can take to enable,
-                e.g., "you must join an organization before building a pitch" */}
-    </div>
-  );
-};
+//   // condition for the component to render - this might be because of the state of user data or something else
+//   function shouldRender() {
+//     return userData;
+//   }
+
+//   return shouldRender() ? (
+//     <div>Value if the component is available.</div>
+//   ) : (
+//       <StyleCard>
+//             <div>
+
+//       Value if the component is not available.
+//       {/* Normally blank but might have a step the user can take to enable,
+//                 e.g., "you must join an organization before building a pitch" */}
+//     </div></StyleCard>
+
+//   );
+// };
```

Comment:
Why did this whole file get commented out? Not that it should be doing anything in particular, just curious.